pkg/overlay/cache.go

```diff
@@ -92,6 +90,16 @@ type UpdateRequest struct {
 	NodeID       storj.NodeID
 	AuditSuccess bool
 	IsUp         bool
+
+	// n.b. these are set values from the satellite.
+	// They are part of the UpdateRequest struct in order to be
+	// more easily accessible in satellite/satellitedb/overlaycache.go.
+	AuditLambda  float64
```

Comment:
I think that I'm missing something, why do we store the lambadas and the weights for each node?

Shouldn't those to be equal for each node?

I've read the comment but I don't fully understand and thinking if there is a way to access to them without having to store them for each node.

Reply:
They should be equal for for each node. The issue is we couldn't find an easy way to set those values on the satellite config and access them from the satellitedb (`satellite/satellitedb/overlaycache.go`) without passing them down from the overlay cache (`pkg/overlay/cache.go`)

Reply:
What about passing them to [`pkg/overlay.NewCache`](https://godoc.org/storj.io/storj/pkg/overlay#NewCache) and satellite can provide them from the [`Config`](https://godoc.org/storj.io/storj/satellite#Config) when the [_Overlay cache_ is instantiated](https://github.com/storj/storj/blob/master/satellite/peer.go#L263)

Reply:
That is already happening. We updated the config with values in `pkg/overlay/config.go`, and the `Node` struct in that file is being passed into `NewCache`. Then, we pass those values (stored in `cache.preferences` down into the db layer

Reply:
yeah the issue is that the two cache structs are different. foiled by wrapper functions once again!

Reply:
I don't get it then why we have to store those values in the DB, but anyway, if you're clear about it it's fine to me as I'm less familiar than you with the code base.

Reply:
we're not storing lambda or weight in the db, we're only storing alpha and beta in the db. we're passing lambda and weight to the "db-level" to use to calculate the reputation/determine if disqualified.

Reply:
oh, I see, I got confused as my first question was referencing to that and I didn't get this response :sweat_smile: 

:thinking: I prefer to have those calculations on the business layer, but I guess that you're doing in the DB layer because you do it in the same `Update..` function rather than having to call `Get` and `Update`

Reply:
right, I also agree it's not ideal but hopefully we'll refactor the overlay cache to separate these kinds of concerns soon.